packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/attribute-props-transform/expected.js

```diff
@@ -31,18 +31,18 @@ function tmpl($api, $cmp, $slotset, $ctx) {
         ariaColSpan: "tranformed",
         ariaControls: api_scoped_id("tranformed"),
         ariaCurrent: "tranformed",
-        ariaDescribedBy: api_scoped_id("tranformed"),
+        ariaDescribedby: api_scoped_id("tranformed"),
         ariaDetails: api_scoped_id("tranformed"),
         ariaDisabled: "tranformed",
-        ariaErrorMessage: api_scoped_id("tranformed"),
+        ariaErrormessage: api_scoped_id("tranformed"),
         ariaExpanded: "tranformed",
-        ariaFlowTo: api_scoped_id("tranformed"),
+        ariaFlowto: api_scoped_id("tranformed"),
         ariaHasPopup: "tranformed",
         ariaHidden: "tranformed",
         ariaInvalid: "tranformed",
         ariaKeyShortcuts: "tranformed",
         ariaLabel: "tranformed",
-        ariaLabelledBy: api_scoped_id("tranformed"),
+        ariaLabelledby: api_scoped_id("tranformed"),
```

Comment:
This makes me nervous; we should probably keep the existing behavior for backwards compat